public/css/main.css

```diff
@@ -12,6 +12,21 @@
     height: 725px;
     float: right;
     width: 25%;
+    border: solid; 
+}
+
+.toppanel {
+    width: 100%;
+    height: 330px;
```

Comment:
Can we make `.toppanel` and `.bottompanel` responsive – if you're on a small screen, you won't be able to see these panels. Look into `vh` – something like `40vh` would be a good thought